### ISSUE TITLE
Returning array of structs

### ISSUE THREAD

**Issue by landersson** [LinkTed/dbus-async-derive#2]
Trying to implement a method with return value signature 'a(qtss)', using the return value:
```
-> Result<Vec<(u16, u64, String, String)>, (Error, String)>
```

I get the following error:

```
8 | #[derive(Handler)]
  |          ^^^^^^^ expected enum `Value`, found struct `Vec`
  |
  = note: expected struct `Vec<Value>`
             found struct `Vec<Vec<Value>>`
```

What return type should I use for the rust method to make it work?

**Comment by LinkTed** [comment]
It seems that there was a bug for struct return type. I fixed it in the new version `2.0.2` (see `examples/container.rs`).

**Comment by landersson** [comment]
Thanks for the speedy response! Seems to work now.
